apps/server/src/modules/files-storage/controller/api-test/files-storage-download-upload.api.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { createMock, DeepMocked } from '@golevelup/ts-jest';
 import { EntityManager } from '@mikro-orm/mongodb';
 import { ICurrentUser } from '@modules/authentication';
-import { JwtAuthGuard } from '@modules/authentication/guard/jwt-auth.guard';
+import { JwtAuthGuard } from '@src/modules/authentication/guard/jwt-auth.guard';
```

Comment:
Please use @ modules the @ src will removed soon.

Reply:
Not touched by us (git merge accident). Now set back to main: 6f30244c61a5e93e222e0fb8f3f7246b07bf951a